.semaphore/semaphore.yml

```diff
@@ -65,15 +65,19 @@ blocks:
       prologue:
         commands:
           - checkout && cd ~/git/flowcrypt-ios/ && cache restore && make dependencies && cache store
-          - sem-version node 16 && mkdir .custom-npm && cd .custom-npm && npm install npm@8.1.2 && cd ..
-          - ( cd Core && cache restore core-npm && ../.custom-npm/node_modules/.bin/npm install && cache store core-npm node_modules )
+          - npm install -g npm@8.11.0
+          - ( cd Core && cache restore core-npm && npm install && cache store core-npm node_modules )
+          - ( cd appium && cache restore appium-npm && npm install && cache store appium-npm node_modules )
       jobs:
         - name: TypeScript tests + code quality
           commands:
             - cd Core
-            - ../.custom-npm/node_modules/.bin/npm test
-            - ../.custom-npm/node_modules/.bin/npm run tslint
-            - ../.custom-npm/node_modules/.bin/npm run eslint
+            - npm test
+            - npm run tslint
+            - npm run eslint
+            - cd ..
+            - cd appium
```

Comment:
can be one command as `cd ../appium` which will actually run faster on semaphore, especially when their servers are congested. There is overhead for each semaphore command